pkgs/tools/system/ddrescue/default.nix

```diff
@@ -15,13 +14,6 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ lzip ];
 
-  doCheck = hostPlatform == buildPlatform;
-
-  ${if hostPlatform != buildPlatform then "crossPlatforms" else null} = [ ];
```

Comment:
This should have been `configurePlatforms`, but that wasn't shows that isn't needed after all!